pkgs/development/python-modules/wxPython/3.0.nix

```
@@ -37,7 +53,9 @@ buildPythonPackage rec {
 
   nativeBuildInputs = [ pkgconfig ]
     ++ (lib.optionals (!stdenv.isDarwin) [ wxGTK libX11 ])
-    ++ (lib.optionals stdenv.isDarwin [ wxmac darwin.apple_sdk.frameworks.Cocoa ]);
+    ++ (lib.optionals stdenv.isDarwin ([ wxmac ] ++ darwinFrameworks));
```

```suggestion
    ++ lib.optional stdenv.isDarwin wxmac;
```

Not sure about linux but for darwin only wx needs to be a native input because of `wx-config`.